### DISCUSSION TITLE
Read And Write Duckdb File Stored On S3 Bucket

### DISCUSSION THREAD

**Discussion by arunksoman** [duckdb/duckdb#10029] [General]
Is there is any way to read and write data on duckdb file stored on s3 bucket directly? I know there is an option to read parquet file stored on s3.

**Comment by soumilshah1995** [comment]
any updates on thread ?

**Comment by robotslacker** [comment]
You can try install some FUSE library, then open database with POSIX format.
Such as: juicefs.  It will create a local POSIX file system based on s3 bucket.
